dist/marionette.gridview.js

```diff
@@ -77,15 +82,15 @@
     initialize: function(options) {
       options = options || {};
       options.gsOptions = options.gsOptions || {};
-      this.autoSave = options.autoSave;
   
-      if (!_.isUndefined(options.autoPos)) {
+      if (_.isUndefined(options.autoPos)) {
         options.autoPos = true;
       }
       if (!options.collection) {
         throw new Error('Missing collection inside initialization options');
       }
       this.options = options;
```

Comment:
this is misleading, options was really part of Backbone.View before. Please use something else as `settings`